<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>725</width>
    <height>516</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Start url:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="leStartUrl">
       <property name="text">
        <string>http://developer.alexanderklimov.ru/android/</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Login:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="leLogin">
       <property name="text">
        <string>denisgamza@yandex.ru</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Password:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lePassword">
       <property name="text">
        <string>azmam</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pbAuth">
       <property name="text">
        <string>Auth</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QPushButton" name="pbGoToStartUrl">
       <property name="text">
        <string>Go to Start Url</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pbPrintPDF">
       <property name="text">
        <string>Pring PDF</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTextEdit" name="txtLog"/>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>pbAuth</sender>
   <signal>clicked()</signal>
   <receiver>Widget</receiver>
   <slot>pbAuthClick()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>686</x>
     <y>58</y>
    </hint>
    <hint type="destinationlabel">
     <x>716</x>
     <y>37</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>pbAuthClick()</slot>
 </slots>
</ui>
